R/easystats_model_dashboard.R

```diff
@@ -22,17 +22,20 @@
 #'   RMarkdown template file. By default, package uses the template shipped with
 #'   the package installation (`inst/templates/easydashboard.Rmd`).
 #'
+#' @inheritParams parameters::model_parameters.default
+#'
 #' @examples
 #' if (FALSE) {
 #'   mod <- lm(wt ~ mpg, mtcars)
-#'   easystats_model_dashboard(mod)
+#'   model_dashboard(mod)
 #' }
 #'
 #' @export
-easystats_model_dashboard <- function(model,
-                                      output_file = "easydashboard.html",
-                                      output_dir = getwd(),
-                                      rmd_dir = system.file("templates/easydashboard.Rmd", package = "easystats")) {
+model_dashboard <- function(model,
+                            output_file = "easydashboard.html",
+                            output_dir = getwd(),
+                            rmd_dir = system.file("templates/easydashboard.Rmd", package = "easystats"),
+                            exponentiate = FALSE) {
```

Comment:
If we start adding individual parameters like these, we would end up with hundreds of parameters for this function. 

We instead need to have a single parameter per underlying function that we pass to the respective function. 

E.g.

```r
model_dashboard <- function(model,
                            output_file = "easydashboard.html",
                            output_dir = getwd(),
                            rmd_dir = system.file("templates/easydashboard.Rmd", package = "easystats"),
                            parameters.args = list(exponentiate = FALSE, ci = 0.90),
                            performance.args = list(metrics = c("AIC", "AICc", "BIC"), verbose = FALSE) {}
```

Reply:
@strengejacke Please see my comment here.

Reply:
yes, probably makes sense.